.travis.yml

```diff
@@ -65,10 +60,9 @@ notifications:
   email: false
 
 before_install:
-  - if [[ $EXECUTE_TEST_COVERALLS != 'true' ]]; then phpenv config-rm xdebug.ini || return 0 ; fi
+  - if [[ $TEST_COVERAGE != 'true' ]]; then phpenv config-rm xdebug.ini || return 0 ; fi
   - travis_retry composer self-update
-  - if [[ $EXECUTE_TEST_COVERALLS == 'true' ]]; then composer require --dev --no-update satooshi/php-coveralls ; fi
-  - composer require --no-update "phpunit/phpunit:$PHPUNIT_VERSION"
+  - if [[ $TEST_COVERAGE == 'true' ]]; then composer require --dev satooshi/php-coveralls $COMPOSER_ARGS; fi
```

Comment:
It would be nice to have global env:
```COVERAGE_DEPS="satooshi/php-coveralls"```
and I think this should be in section `install`, please compare your travis configuration for example with:
https://github.com/zendframework/zend-expressive/blob/master/.travis.yml

Would be nice to have it consistent.